mass/core/channel.py

```diff
@@ -629,12 +629,12 @@ def __init__(self, pulserec_dict, tes_group=None, hdf5_group=None, invert_data=F
         self.times = None
         self.subframecount = None
 
-        self.number_of_rows = None
-        self.row_number = None
-        self.number_of_columns = None
-        self.column_number = None
-        self.subframe_divisions = None
-        self.subframe_offset = None
+        self.number_of_rows = 1
```

Comment:
This set of changes seems totally uneccesary and actually bad. We found this error because mass crashed when something went wrong, by giving default values instead of `None` values to all these parameters, we increase the likelyhood mass will do something silently wrong instead of crashing. These changes aren't needed to fix the bug at hand.